iris/docs/version_switch.js

```diff
@@ -11,7 +11,7 @@
     // nb. Also include the latest version explictly as well as under "latest",
     // so that users can get to the versioned (non-latest) URL.
 
-    'v2.2': '2.2',
```

Comment:
As before, this is updated so that the version switcher at the top right hand corner selects the 2.2.1 docs when you pick 2.2